README.md

```diff
@@ -17,8 +34,17 @@ To deploy, create a playbook which looks like this:
             control: "{{ inventory_hostname in groups['cluster_control'] }}"
             batch: "{{ inventory_hostname in groups['cluster_batch'] }}"
             runtime: true
+          openhpc_slurm_service_enabled: true
+          openhpc_slurm_service: slurmd
+          openhpc_slurm_control_host:
+          openhpc_slurm_partitions: []
```

Comment:
Here's an example of what `openhpc_slurm_partitions` variable looks like: https://github.com/stackhpc/p3-appliances/blob/master/config/openhpc.yml